firebase/functions/src/comments.ts

```diff
@@ -85,4 +108,20 @@ function addToFIreBase(data: any, collection: any) {
       });
   });
 }
-//943.05
+
+function update(data: any, collection: any,id:any) {
+  return new Promise((resolve, reject) => {
+      const userRef = db.collection(collection).doc(id)
+      userRef.update(data).then(function () {
+          resolve({
```

Comment:
We could save a log about the document updates:
```suggestion
          console.log(`Document updated /${collection}/${id} with data:`, data)
          resolve({
```